db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
```

Comment:
- [ ] This seeds.rb file is empty, resulting in new installs having no houses and an eternal loading screen.

![image](https://user-images.githubusercontent.com/57421823/203852526-78b5d480-e54d-4042-9f1e-a37de6785b35.png)

Although it is of course possible to just add houses through the console, it is very unlikely that your potential employers have the drive or time to do it manually in order to check out your app.  

Kindly add at least 1-3 houses, so they can see the carrousel and test your app locally. 💪